### ISSUE TITLE
Add workflow that allows manual allow/disallow with a spec path

### ISSUE THREAD

**Issue by CBonade** [department-of-veterans-affairs/va.gov-team#66898]
## Issue Description
A manual workflow to properly enable or disable a unit test or an e2e test will be very helpful for resolving support issues, unblocking errors and solving other situations. This workflow should be attached to the qa-standards-dashboard-data repository and should have some kind of permissions restriction if possible.

---
## Tasks
- [ ] Refactor complex functions to be able to be called separately, such as the ones to construct the queries for BigQuery.
- [x] Implement workflow that takes in workflow args on a dispatch and makes the desired adjustment using the same functions that power the allow lists.

## Acceptance Criteria
- [x] Manual adjustments to the allow lists can be easily made without going directly through BigQuery.

**Comment by CBonade** [comment]
https://github.com/department-of-veterans-affairs/qa-standards-dashboard-data/actions/workflows/allow-list-adjustment.yml 

This works and can be launched from here!

**Comment by CBonade** [comment]
Refactoring the complex functions is going to involve a lot of testing of the main process to ensure no breakage and should be done as a part of a separate task once we have the full scope of adjustments that need to be made to our process after this has been launched for a bit.
